app/javascript/tests/layout/Header.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
Do you have a the bandwidth to add a full test suite for this? It's not a huge component so the the tests I would add are
- Properly renders all main components (everything exists and has the right text)
- Active tabs/hide showing of tabs (will have to change user)
- Proper rendering of the help links (all options) - what you currently have
- Shows correct navbar options if usa admin (will have to change the user)
- Defining banner message properly renders it

I might have missed some cases but you get the gist.  If you need help or examples let me know

Reply:
@sgober This is my first pass at adding tests for the Header component. If you have time time, it would be great to have some interim feedback about the approach/style. Thank you! 

[ef5418a](https://github.com/SaraAlert/SaraAlert/pull/1243/commits/ef5418ac9bf3f0c7f52da2f3dc22445a3b873ffa)

Reply:
Also @sgober I haven't updated the tracking yet because I'm still getting error messages when trying to access (latest was a 405)